src/components/card/card.tsx

```diff
@@ -73,9 +73,17 @@ const Card: React.FunctionComponent<CardProps> = ({
                     hasThumbnail(variant, pillCategory) && (
                         <div sx={thumbnailWrapperStyles(variant, pillCategory)}>
                             <Image
-                                alt={thumbnail.alt || 'alt not provided'}
+                                alt={
+                                    pillCategory === 'Podcast'
+                                        ? 'Play Button'
+                                        : thumbnail.alt || 'alt not provided'
+                                }
                                 loader={thumbnailLoader}
-                                src={thumbnail.url}
+                                src={
+                                    pillCategory === 'Podcast'
+                                        ? '/developer/play-button.svg'
```

Comment:
Missed this one; you can use the getURL utility here

Reply:
ahh will do, I was only thinking about that for hrefs but definitely good to use it here too.